src/components/DependencyGraph/index.js

```diff
@@ -67,66 +70,44 @@ export default class DependencyGraphPage extends Component {
     this.props.fetchDependencies();
   }
 
-  handleGraphTypeChange(graphType) {
-    this.setState({ graphType });
-  }
+  handleGraphTypeChange = graphType => this.setState({ graphType });
 
   render() {
     const { nodes, links, error, dependencies, loading } = this.props;
     const { graphType } = this.state;
     if (loading) {
-      return (
-        <div className="m1">
-          <div className="ui active centered inline loader" />
-        </div>
-      );
+      return <LoadingIndicator className="u-space-top-vast" centered />;
     }
     if (error) {
-      return <NotFound error={error} />;
+      return <ErrorMessage className="u-space" error={error} />;
```

Comment:
Curious what `u-space` does? Is it a utility?

I notice that we removed [Basscss](http://basscss.com/). IMO, I think we should use basscss as a CSS utility library since its small (~2kb), low on side effects and documented.